File: script/python/game/util/logger.py

import Logger


def log_info(fmt, *p):
    Logger.logInfo("$[PyLog] " + fmt.format(*p))


def log_error(fmt, *p):
    Logger.logError("$[PyLog] " + fmt.format(*p))


def log_debug(fmt, *p):
    Logger.logDebug("$[PyLog] " + fmt.format(*p))


def log_warn(fmt, *p):
    Logger.logWarn("$[PyLog] " + fmt.format(*p))

File: script/lua/util/logger.lua

logger = {}

function logger.logInfo(fmt, ...)
    local msg = string.format(fmt, ...)
    Logger.log_info("[Lua] " .. msg)
end

function logger.logError(fmt, ...)
    local msg = string.format(fmt, ...)
    Logger.log_error("[Lua] " .. msg)
end

function logger.logDebug(fmt, ...)
    local msg = string.format(fmt, ...)
    Logger.log_debug("[Lua] " .. msg)
end

function logger.logWarn(fmt, ...)
    local msg = string.format(fmt, ...)
    Logger.log_warn("[Lua] " .. msg)
end
